ios/CurrentLevelReviewTimeItem.swift

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020 David Sansome
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+@objc
+class CurrentLevelReviewTimeItem: NSObject, TKMModelItem {
+  let services: TKMServices
+  let currentLevelAssignments: [TKMAssignment]
+
+  @objc init(services: TKMServices, currentLevelAssignments: [TKMAssignment]) {
+    self.services = services
+    self.currentLevelAssignments = currentLevelAssignments
+  }
+
+  func createCell() -> TKMModelCell! {
+    return CurrentLevelReviewTimeCell(style: .value1,
+                                      reuseIdentifier: String(describing: CurrentLevelReviewTimeCell.self))
+  }
+}
+
+class CurrentLevelReviewTimeCell: TKMModelCell {
+  private var services: TKMServices?
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+    selectionStyle = .none
+    detailTextLabel?.textColor = TKMStyle.Color.label
+  }
+
+  required init?(coder _: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  override func update(with baseItem: TKMModelItem!) {
+    let item = baseItem as! CurrentLevelReviewTimeItem
+    var reviewDates = [Date]()
+
+    for assignment in item.currentLevelAssignments {
+      if let reviewDate = assignment.reviewDate {
+        if !assignment.isLessonStage {
+          reviewDates.append(reviewDate)
+        }
+      }
+    }
+
+    // Sort the list of dates
+    reviewDates.sort()
+
+    if let lastReviewDate = reviewDates.last {
```

Comment:
```suggestion
    if let lastReviewDate = reviewDates.first {
```
Shouldn't reviewDates.first be used at this point? 
In my case, next current level review for radicals is in 9 hours, but it says that will be in 21 hours.

![IMG_0945](https://user-images.githubusercontent.com/1460558/71961051-ce3c5e00-31f6-11ea-9fb9-7eb5f43aed24.PNG)

![IMG_0946](https://user-images.githubusercontent.com/1460558/71961052-ce3c5e00-31f6-11ea-9132-77ca01e2a457.PNG)

In any case, thank you very much for this awesome feature!

Reply:
Fixed it, thanks! I'm at level 19.5 right now, so all of my current level items are at Lesson. 😉

Reply:
You're welcome! 😄